_layouts/media.html

```diff
@@ -105,6 +105,7 @@
 
 var talks = [
 	{ id: "wSpPjUPIEVU", title: "OSDC 2015 Foreman in your data center", author: "Stephen Benjamin" },
+	{ id: "b18Us1nAKMA", title: "PuppetCamp 2015: Selling management and developers on Puppet", author: "David Patterson"},
```

Comment:
@Dan33l No need to create a new pull request, fix the mistake, `git commit --amend`, and --force push against the same branch. Thanks!